docs/sphinx/users/octave/index.txt

```diff
@@ -0,0 +1,41 @@
+GNU Octave
+==========
+
+`GNU Octave <http://www.octave.org>`_ is a high-level interpreted language,
+primarily intended for numerical computations.
+Being an array programming language, it is naturally suited for image
+processing and handling of N dimensional datasets.
+Octave is distributed under the terms of the GNU General Public License.
+
+The Octave language is Matlab compatible so that programs are easily
+portable.
+Indeed, the Octave bioformats package is exactly the same as Matlab's,
```

Comment:
The MATLAB toolbox has the bioformats_package.jar bundled with it so this may be slightly misleading even though I accept that does only affect the installation.